libs/barista-components/schematics/src/ng-add/__snapshots__/ng-add.spec.ts.snap

```diff
@@ -468,42 +468,42 @@ Object {
     "@angular/animations": "^8.2.12",
     "@angular/cdk": "^8.2.12",
     "@angular/common": "^8.2.12",
-    "@angular/compiler": "~10.0.5",
+    "@angular/compiler": "~10.1.6",
     "@angular/core": "^8.2.12",
-    "@angular/forms": "~10.0.5",
-    "@angular/platform-browser": "~10.0.5",
-    "@angular/platform-browser-dynamic": "~10.0.5",
-    "@angular/router": "~10.0.5",
+    "@angular/forms": "~10.1.6",
+    "@angular/platform-browser": "~10.1.6",
+    "@angular/platform-browser-dynamic": "~10.1.6",
+    "@angular/router": "~10.1.6",
```

Comment:
Bumping the NX version causes newer versions to be installed, so I had to update the snapshot to work around tests failing.